scripts/im_plot.sl

```diff
@@ -30,14 +32,18 @@ echo ___im plot___
 
 python $gmsim/workflow/scripts/cybershake/add_to_mgmt_queue.py $MGMT_DB_LOC/mgmt_db_queue $SRF_NAME IM_plot running $SLURM_JOB_ID
 
-COMPS = `cat $CSV_PATH |cut  -d , -f 2 |sort |uniq` # find what components are used
-COMPS=($(echo $COMPS |tr " " "\n")) # convert to an array of strings
-COMPS_LEN=${#COMPS[@]} # length of the array
-for (( i=1;i<$COMPS_LEN;i++ )); #${COMPS[0]} is the header "component"
+# check rotd50 was found in .csv
+if [[ " ${COMPS[*]} " == *" rotd50 "* ]]; 
+then 
+    comps_to_plot=(geom rotd50)
+else 
+    comps_to_plot=(geom)
+fi
+
+for comp in ${comps_to_plot[@]};
 do 
-    COMP=${COMPS[$i]}
-    OUTPUT_XYZ_DIR=$3/$COMP
-    res=`python $gmsim/visualization/im/spatialise_im.py $CSV_PATH $STATION_FILE_PATH --out_dir $OUTPUT_XYZ_DIR -c $COMP`
+    OUTPUT_XYZ_DIR=OUTPUT_XYZ_PARENT_DIR/$comp
```

Comment:
`OUTPUT_XYZ_DIR=$OUTPUT_XYZ_PARENT_DIR/$comp` ? Are you missing a `$`

Just for sanity have you tested this with the automated workflow?

Reply:
Good catch. No I haven't tested it. I was going to ask you to test assuming you already have a setup.

Reply:
I don't have something that would easily be useable. I only really have the CS dir.

Reply:
ok - i'll take care of it